packages/utils/src/utils.ts

```diff
@@ -70,3 +73,79 @@ export function distinct<T>(array: Array<T>): Array<T> {
 export function nodeName(node: Node): string {
     return node.nodeName.toUpperCase();
 }
+
+/**
+ * Get the target VNode or Format's style attribute as a record of style name to
+ * style value.
+ *
+ * @param target
+ */
+export function getStyles(target: VNode | Format): Record<string, string> {
+    const stylesArray = (target.attributes.style as string || '')
```

Comment:
I'm surprised the type isn't already `string` ?

Reply:
```ts
Attributes: Record<string, string | Record<string, string>;
```
It doesn't know that `style` was not a `Record<string, string>`.